spec/lita/services/market_manager_spec.rb

```diff
@@ -283,6 +289,20 @@ def setup_lunchers
         expect(lunch_assigner.winning_lunchers_list).to include(fdom.mention_name)
         expect(lunch_assigner.winning_lunchers_list).not_to include(andres.mention_name)
       end
+
+      it 'uploads transaction' do
+        subject.execute_transaction
+        expect(sh_manager).to have_received(:insert_new_row).with(
+          [
+            Time.now.strftime('%Y-%m-%d'),
```

Comment:
Do not use Time.now without zone. Use one of Time.zone.now, Time.current, Time.now.in_time_zone, Time.now.utc, Time.now.getlocal, Time.now.xmlschema, Time.now.iso8601, Time.now.jisx0301, Time.now.rfc3339, Time.now.httpdate, Time.now.to_i, Time.now.to_f instead.